siriuspy/siriuspy/devices/ids.py

```diff
@@ -1657,15 +1657,15 @@ def set_center_offset(self, center_offset, timeout=None):
 
     def set_center_mode(self, mode, timeout=None):
         """Set ID center mode True or False."""
-        if type(mode) != bool:
+        if type(mode) is not bool:
```

Comment:
@Gabrielrezende-asc , what does dis change fix? I tested both syntaxes here and both work.

Either way, the best way to check types is using `isinstance`, right?